Userland/Libraries/LibWeb/DOM/Event.cpp

```diff
@@ -41,6 +41,20 @@ Event::Event(HTML::Window& window, FlyString const& type, EventInit const& event
 {
 }
 
+void Event::visit_edges(Visitor& visitor)
+{
+    visitor.visit(m_target.ptr());
```

Comment:
We need `Base::visit_edges(visitor)` here to make sure the properties of the base class are also visited.